travis/build-docker.sh

```diff
@@ -5,14 +5,9 @@ set -xe
 old_cwd=$(readlink -f .)
 here=$(readlink -f $(dirname "$0"))
 
-DOCKERFILE="$here"/Dockerfile.centos6
-IMAGE=linuxdeploy-build-centos6
-
-if [ "$ARCH" == "i386" ]; then
-    DOCKERFILE="$DOCKERFILE"-i386
-    IMAGE="$IMAGE"-i386
-fi
+DOCKERFILE="$here/Dockerfile.$ARCH"
```

Comment:
If you now require `$ARCH` to be set, please add a check above to make sure the value is explicitly set. The script must run locally, too. If I have to `env ARCH=x86_64 travis/build-docker.sh`, that's fine, but only if there's a nice error message if `$ARCH` is not set.